vidarr-pluginapi/src/main/java/ca/on/oicr/gsi/vidarr/OutputProvisioner.java

```diff
@@ -77,7 +79,7 @@ interface ResultVisitor {
     void file(String storagePath, String md5, long size, String metatype);
 
     /**
-     * The output is a reference to another system
+     * The output is a reference to another system *
```

Comment:
\* ?

Reply:
That's straight from the code formatting plugin.

Reply:
Is the google code formatter yet configurable to let comments be? it also mangles things like example json unless you know to add `<pre>` to your comment, which isn't obvious

Reply:
The `google-java-format` tool is extremely opinionated about [whether and how it can be configured](https://github.com/google/google-java-format/wiki/FAQ#i-just-need-to-configure-it-a-bit-differently-how). Code snippet mangling in the absence of `<pre>` looks to be consistent Javadoc behaviour, as opposed to the formatter's doing, so I that might be why the option to ignore comment formatting isn't toggle-able in the format tool.